### PR TITLE
Announcement UI

### PR DESCRIPTION
**Pull request by mayethsc**

### CODE REVIEW

File: application/modules/hirer/models/AnnouncementModel.php [hunk 1]
```diff
@@ -25,10 +31,13 @@ public function getLatest($subscriptionTo, $subscriptionType, $approved)
             'status' => 'P',
             'test' => 0,
             'end_date <>' => '0000-00-00',
-            'start_date <=' => $sqltoday,
-            'end_date >=' => $sqltoday
+            'start_date <=' => $sqltoday
```
Comment (princeryansy) [inline application/modules/hirer/models/AnnouncementModel.php]: can we drop using $sqltoday variable and use $dateNow instead or other more appropriate variable name?

File: application/modules/hirer/views/announcement.php [hunk 2]
```diff
@@ -1,4 +1,39 @@
-<?php
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <div class="card mx-auto">
+                    <div class="card-body">
+                        <div class="text-center">
+                            <span class="badge badge-primary">Announcement</span class="badge badge-primary"><br/>
```
Comment (princeryansy) [inline application/modules/hirer/views/announcement.php]: add static content in lang file.

File: application/modules/hirer/views/announcement.php [hunk 3]
```diff
@@ -1,4 +1,39 @@
...
+                        for ($i = 0;$i < $total; $i++) {
+                            echo '<a href="' . url_hirer('announcement/' . $announcements[$i]['id']. '') . '"><span class="badge badge-light text-danger m-1"> ' . $announcements[$i]['title'] . '</span></a>';
+                        }
+                        ?>
+                    </h6>
+                </div>
+            </div>
+        </div>
+    </div>
+
```
Comment (princeryansy) [inline application/modules/hirer/views/announcement.php]: kindly remove extra spaces and leave a blank line.

File: application/modules/hirer/views/announcement.php [hunk 4]
```diff
@@ -1,4 +1,39 @@
...
+                            <span class="badge badge-primary">Announcement</span class="badge badge-primary"><br/>
+                                <small class="text-muted font-italic"><?php echo nice_date($selected['date_created'], 'M-d-Y'); ?></small><br/>
+                                <h1 class="text-info mt-4"><?php echo $selected['title']; ?></h1>
+                                <hr class="w-25">
+                        </div>
+                        <div class="row">
+                            <div class="col-md-12">
+                                <p><?php echo $selected['story']; ?></p>
+                                <div class="text-right font-weight-bold">
+                                    <p class="blockquote-footer">Regards,</p>
```
Comment (princeryansy) [inline application/modules/hirer/views/announcement.php]: add static contents in lang file.

**Comment by mayethsc** [comment]
@princeryansy , you may now check latest commit. Thanks!

**Comment by mayethsc** [comment]
Approved already by @princeryansy .

**Pull request merged**